goal_src/jak1/engine/ps2/pad.gc

```diff
@@ -156,22 +171,19 @@
     (set! (-> gp-0 cpads 3) (new 'global 'cpad-info 3))
     gp-0))
 
- ;; Allocates enough input frames to support counting 50ms of time at 300fps.
+;; Allocates enough input frames to support counting 50ms of time at 300fps.
 (deftype cpad-history (basic)
   ((button0-abs pad-buttons 15) ;; bitmask of buttons, pressed or not, with history
    (button0-rel pad-buttons 15) ;; bitmask of if button going down.
-  )
-)
+   ))
```

Comment:
All the whitespace changes should just be from running the formatter on `goal_src/jak1`

Reply:
Looks like lint is complaining that this changed line numbers for a handful of `;; og:preserve-this` comments